steamos-waydroid-installer.sh

```diff
@@ -432,44 +326,8 @@ else
 	echo -e $PASSWORD\n | sudo -S sed -i "s/ro.hardware.gralloc=.*/ro.hardware.gralloc=minigbm_gbm_mesa/g" /var/lib/waydroid/waydroid_base.prop
 
 	# lets change the fingerprint so waydroid shows up as a Pixel 5 - Redfin
-	echo -e "$current_password\n" | sudo -S tee -a /var/lib/waydroid/waydroid_base.prop > /dev/null <<'EOF'
-
-##########################################################################
-# controller config for udev events
-persist.waydroid.udev=true
-persist.waydroid.uevent=true
-
-# disable root
-ro.adb.secure=1
-ro.debuggable=0
-ro.build.selinux=1
-
-##########################################################################
-### start of custom build prop - you can safely delete if this causes issue
-
-ro.product.brand=google
-ro.product.manufacturer=Valve
-ro.system.build.product=redfin
-ro.product.name=redfin
-ro.product.device=redfin
-ro.product.model=Steam Deck
-ro.system.build.flavor=redfin-user
-ro.build.fingerprint=google/redfin/redfin:11/RQ3A.211001.001/eng.electr.20230318.111310:user/release-keys
-ro.system.build.description=redfin-user 11 RQ3A.211001.001 eng.electr.20230318.111310 release-keys
-ro.bootimage.build.fingerprint=google/redfin/redfin:11/RQ3A.211001.001/eng.electr.20230318.111310:user/release-keys
-ro.build.display.id=google/redfin/redfin:11/RQ3A.211001.001/eng.electr.20230318.111310:user/release-keys
-ro.build.tags=release-keys
-ro.build.description=redfin-user 11 RQ3A.211001.001 eng.electr.20230318.111310 release-keys
-ro.vendor.build.fingerprint=google/redfin/redfin:11/RQ3A.211001.001/eng.electr.20230318.111310:user/release-keys
-ro.vendor.build.id=RQ3A.211001.001
-ro.vendor.build.tags=release-keys
-ro.vendor.build.type=user
-ro.odm.build.tags=release-keys
-
-### end of custom build prop - you can safely delete if this causes issue
-##########################################################################
-EOF
-	
+	{ echo -e "$current_password\n" ; cat extras/waydroid_base.prop ; } | sudo -S tee -a /var/lib/waydroid/waydroid_base.prop
```

Comment:
this is slightly hacky as two inputs are needed since this uses `tee -a` to append and not just overwrite (unlike the others that were all plain `tee`). this syntax follows [this Unix SE answer](https://unix.stackexchange.com/a/391820/152866)